### ISSUE TITLE
How to make date or calendar inputs?

### ISSUE THREAD

**Issue by hugoruscitti** [adfinis-sygroup/ember-validated-form#6]
Hi!, i want to create a form with on date field, but i don't understand how to represent the field to show a date or allow to popup a calendar. I used before an addon called ember-pikaday, but seems to be impossible to integrate in a form made with ember-validated-form.

any advice?

**Comment by czosel** [comment]
Hi, thanks for your bug report!

Getting unsupported input types to work involves two stages:
1. Simply making the input "work", i.e. making sure the data-binding works 
2. Integrating the input in the validation workflow, i.e. showing error messages on focus out, ...

In you're just trying to get step 1 to work, I released version 0.1.5 of this addon that exposes the form's changeset as `f.model`, which allows you to write something like this in your template:
```Handlebars
{{pikaday-input onSelection=(action (mut f.model.birthday))}}
```
I also added this example to the [README](https://github.com/adfinis-sygroup/ember-validated-form#other-input-elements).

If you're out for step 2, there is currently no "easy way to do this". You'll basically have to reimplement the validation logic for the datepicker by binding to focosOut events etc. - It would be nice to offer some kind of integration for this too, but I'm not sure if it's possible/feasible.

Hope this helps!
